flink-runtime-web/web-dashboard/src/app/pages/overview/statistic/overview-statistic.component.less

```diff
@@ -29,10 +29,62 @@ nz-card {
   }
 }
 
+.taskslots {
+  display: flex;
+}
+
 .total {
   color: @success-color;
   font-size: 36px;
   line-height: 38px;
+  flex: 1
+}
+
+.task-manager-avatar {
+  overflow: hidden;
+  display: inline-block;
+}
+
+.task-slot-percentage-bar {
+  float: right;
+  flex: 4;
+  margin-top: 16px;
+}
+
+.task-manager-percentage-bar ::ng-deep .ant-progress {
```

Comment:
try something like this (need additional css to change the left and top position)
.task-manager-percentage-bar {
  transform: rotate(270deg);
  display: inline-block;
}